templates/cemanager.yaml

```diff
@@ -90,6 +90,9 @@ spec:
           ports:
             - name: tcp-cemanager
               containerPort: 8080
+          securityContext:
+            allowPrivilegeEscalation: false
+            readOnlyRootFilesystem: {{ .Values.cemanager.securityContext.readOnlyRootFilesystem }}
```

Comment:
With these changes I think we can move this linter setting from exclude to include: https://github.com/cdr/enterprise-helm/blob/766cc6d061e78c2c3005c0b52d65e8df5bdd2b27/kube-linter.yaml#L19